app/lib/core/view_models/account_edit_view_model.dart

```diff
@@ -6,13 +8,48 @@ import 'package:app/core/services/service_locator.dart';
 import 'package:app/core/view_models/base_view_model.dart';
 import 'package:app/ui/common/view_state.dart';
 import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:permission_handler/permission_handler.dart';
+import 'package:uuid/uuid.dart';
 
 class AccountEditViewModel extends BaseViewModel {
   final DatabaseService _databaseService = locator<DatabaseService>();
   final NavigationService _navigationService = locator<NavigationService>();
   final DialogService _dialogService = locator<DialogService>();
 
+  final _picker = ImagePicker();
+
   Transporter _account;
+  File _selectedImage;
+  File get selectedImage => _selectedImage;
+  String _imageUrl;
+  String get imageUrl => _imageUrl;
+
+  Future<void> selectImage() async {
+    await Permission.photos.request();
```

Comment:
I thought we already had permissions covered for `ImageScreen` @mansiPaTeLL did `ImagePicker` not need this?

You could use `ImageScreen` still here, the screen can return the image picked through [`Navigator.pushNamed`](https://flutter.dev/docs/cookbook/navigation/returning-data)

Reply:
I checked and confirmed that Image screen did not use [Permission handler plugin](https://pub.dev/packages/permission_handler).

Reply:
How the heck was image picker working before if it didn't have permissions?

Reply:
It looks like in android, you don't need to add anything for permission to work. In ios, however, I had to configure something in the info.plist.

Reply:
@holtzmak @MummenRider you need to configure in iOS and I think in android as well. You can check my commits in the experimental task.